typescript-promises/main.ts

```diff
@@ -0,0 +1,4 @@
+import { takeAChance } from './take-a-chance.js';
+const promise1 = takeAChance('Jeffrey');
+promise1.then((message) => console.log(message));
+promise1.catch((error) => console.log(error));
```

Comment:
`.catch` should not have `promise1` before it. `.catch` is getting chained from the `.then`